js/main.js

```diff
@@ -56,6 +55,7 @@ function handleNewSubmit(event) {
     $entryList.prepend(generateEntryDOM(newEntry));
   }
   switchView('entries');
+  data.view = 'entries';
```

Comment:
If you assign `data.view` in your `switchView` function, you can avoid all this repetition